src/modals/manage-farming-pool-modal/details-card/index.tsx

```diff
@@ -112,15 +115,26 @@ export const DetailsCard: FC<DetailsCardProps> = ({
     };
 
     if (msToVestingEnd > 0) {
+      const modalAnswerAnalyticsProperties = {
+        page: ModalsEnum.ManageFarmingPool,
+        farmId: farm.id,
+        farmContractAddress: farm.contractAddress
+      };
+
       doAfterConfirmation(
         'Your claimable rewards will be claimed and sent to you. But your full rewards will be totally lost and redistributed among other participants.',
         'Claim rewards',
-        claimRewards
+        () => {
+          trackEvent('CLAIM_REWARDS_MODAL_CONFIRM', AnalyticsEventCategory.ButtonPress, modalAnswerAnalyticsProperties);
```

Comment:
pls extract selectors into separate enum

Reply:
Actually, it's not a selector but an event name. Actually, there is no way to assign test IDs to buttons of an alert